### PR TITLE
basic scheduling

### PR DESCRIPTION
**Pull request by EFF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,16 @@ npm install -g lightning-cli
...
+lightning scan <url>
 ```
 _url_: valid url of the website you want to be analyzed.
+
+#### Schedule a recurring scan
+```
+lightning schedule <url> --interval --start
+```
+_url_: valid url of the website you want to be analyzed.
+_--interval_: A time interval in milliseconds. Defaults to one day
```
Comment (osylvain) [inline README.md]: should we add that there's a min value for interval, even if it might be temporary?

File: bin/lightning-scan.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  else {
+    console.log('scanning ', targetUrl, ' ...');
+
+    var requestOptions = {
+      json: {url: targetUrl}
+    };
+
+    // TODO: wrap request in a promise and extract to a index.js module so lightning can be called by via node
+    request.post(LIGHTNING_TASK_URL, requestOptions, function (err, httpResponse, body) {
+      if (err) return console.error('Oups, somethig went wrong while scanning ...', err.message)
```
Comment (osylvain) [inline bin/lightning-scan.js]: somethi **n** g

File: bin/lightning-scan.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+function immediateScan(targetUrl) {
+  if (!targetUrl || typeof targetUrl != "string") console.log('a valid target URL needs to be specified');
+  else {
+    console.log('scanning ', targetUrl, ' ...');
+
+    var requestOptions = {
+      json: {url: targetUrl}
+    };
+
+    // TODO: wrap request in a promise and extract to a index.js module so lightning can be called by via node
```
Comment (osylvain) [inline bin/lightning-scan.js]: yeah would be nice!

File: bin/lightning-scan.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    request.post(LIGHTNING_TASK_URL, requestOptions, function (err, httpResponse, body) {
+      if (err) return console.error('Oups, somethig went wrong while scanning ...', err.message)
+      else if (body.message == 'Endpoint request timed out') { // API GATEWAY TIMES OUT AFTER 10 seconds
+        outputSuccessMessage('Perf scan not complete yet, your results will likely be available at ' + LIGHTNING_WEBAPP_BASE_URL + ' in less than a minute');
+      } else if (body.metrics) {
+        outputSuccessMessage('Perf scan complete, go check your results at ' + LIGHTNING_WEBAPP_BASE_URL);
+        for (metric in body.metrics) {
```
Comment (osylvain) [inline bin/lightning-scan.js]: Could add a verbose arg and display metrics accordingly. Or have them displayed by default and have a silent or quiet arg. Just that it seems annoying to always have a 200 lines output every time I scan. What do you think?

**Comment by osylvain** [comment]
pretty neat! Check the comments but you can merge!

**Comment by EFF** [comment]
I updated the readme, other comments will be fixed in the other branch

**Pull request merged**